web3j/pom.xml

```diff
@@ -50,18 +49,6 @@
       <scope>provided</scope>
     </dependency>
 
-    <dependency>
```

Comment:
So looks like there is some compatibility issue because of which you want to remove support for  embedded EVM. This is fine as long as you remove it completely.

Reply:
okay sure